conventions.md

```diff
@@ -0,0 +1,15 @@
+# Conventions
+
+A basic overview of (semi-)arbitrary conventions we follown in this repository.
+
+## Closed channel error handling
```

Comment:
This actually standardizes only one side of the channel, maybe since you are already at it you could do the same for `next()` returning `None`?

Reply:
In current state channel error handling is quite straight forward, so this file is no longer needed